logging/impl_test.go

```diff
@@ -298,3 +300,146 @@ func TestSublogger(t *testing.T) {
 	assertLogMatches(t, notStdout,
 		`2023-10-30T09:12:09.459Z	INFO	impl.sub	logging/impl_test.go:67	info log`)
 }
+
+func TestLoggingWithFields(t *testing.T) {
```

Comment:
Excellent job on these tests 🧑‍🔧 . Some interesting edge cases covered.

Reply:
I stole them above from a different test lol

Reply:
Quality theft; thanks @dgottlieb (I'm assuming) for the originals.